phac-nml/irida-next
# List samples with transfer dialog

## What does this PR do and why?
Moved the checkboxes into their own column within the group &  project samples tables. 
Added a list of selected samples to the transfer dialog.
Fixes #468.

## Screenshots or screen recordings
![Screenshot from 2024-03-20 15-31-12](https://github.com/phac-nml/irida-next/assets/325703/243ece6d-46a3-4d95-abd2-7c05fe479da8)
![image](https://github.com/phac-nml/irida-next/assets/325703/c31bf83d-e70e-4d45-b2f9-d30590f45645)

## How to set up and validate locally
1. Login to irida-next.
2. Navigate to a project samples page that the user is a maintainer or owner. 
3. Select some samples and click the `Transfer samples` button.
4. Verify the selected samples are listed within the dialog.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
